src/create_disk.sh

```diff
@@ -313,6 +313,10 @@ set prefix=($prefix)/grub2
 configfile $prefix/grub.cfg
 boot
 EOF
+    # Disable shim fallback; install grub as second-stage bootloader for
+    # fallback path instead
+    rm "${target_efi}"/EFI/BOOT/fb*.efi
+    cp "${target_efi}/EFI/${vendor_id}"/grub*.efi "${target_efi}"/EFI/BOOT/
```

Comment:
Hmm, so `shim` just dumbly looks at whatever other binaries there are in `EFI/BOOT` regardless how it's named?

Reply:
Nope, there are [two](https://github.com/rhboot/shim/blob/main/shim.c#L1115-L1116) [cases](https://github.com/rhboot/shim/blob/main/shim.c#L1135) and we're switching to the other one.